Riot/Utils/Tools.m

```diff
@@ -138,4 +138,41 @@ + (NSAttributedString *)setTextColorAlpha:(CGFloat)alpha inAttributedString:(NSA
     return string;
 }
 
++ (NSAttributedString *)attributedStringFromHTML:(NSString *)htmlString withAllowedTags:(NSArray<NSString *> *)allowedTags
```

Comment:
I'm not fan of massive random `Tools` classes. Either we create an `HTMLFormatter` (or `AttributedStringFormatter`) or create extension methods like:
```
extension String {
func formattedHTML(allowedTags: [String], fontSize: CGSize)
}

Reply:
Agreed, I'll add this and the other one in an extension. It's not quite general purpose enough yet, but my goal is to have this used everywhere as MXKEventFormatter and the group home view controller both do this themselves too.

Reply:
I went with the `HTMLFormatter` approach in the end.